vcu/Src/drive_by_wire_mock.c

```diff
@@ -440,6 +440,75 @@ static const CLI_Command_Definition_t mcInitCommandDefinition =
     0 /* Number of parameters */
 };
 
+
+BaseType_t setInverterParameter(char *writeBuffer, size_t writeBufferLength, const char *commandString){
+    uint16_t parameterAddress;
+    uint16_t newValue;
+    BaseType_t status;
+    uint16_t readValue = 0;
+    uint32_t currentState = fsmGetState(&fsmHandle); //returns an index
+    BaseType_t paramLen;
+
+    const char *addressParam = FreeRTOS_CLIGetParameter(commandString, 1, &paramLen);
+    const char *valueParam = FreeRTOS_CLIGetParameter(commandString, 2, &paramLen);
+    
+    sscanf(addressParam, "%hu", &parameterAddress);
+    sscanf(valueParam, "%hu", &newValue);
+
+    if (HV_Power_State == HV_Power_State_On){ 
+        COMMAND_OUTPUT("Error: Car is at high voltage. Operation not permitted\n");
+        return pdFALSE;
+    }
+    else if (currentState == STATE_EM_Enable){ 
+        COMMAND_OUTPUT("Error: Car is in drive mode. Operation not permitted\n");
+        return pdFALSE;
+    } else{
```

Comment:
Stay consistent with your formatting, for else if you put it on a new line but for the else statement you kept it on the same line as your closing bracket. I would simply get rid of the else statement, it's not really needed here.